srcts/src/components/errorConsole.ts

```diff
@@ -246,7 +270,7 @@ class ShinyErrorConsole extends LitElement {
           </svg>
         </button>
       </div>
-      <slot class="content"></slot>`;
+      <slot class="content" @slotchange=${this.dedupeConsoleMessages}></slot>`;
```

Comment:
I don't love that this dedupes after the content has changed, by deduping all of the messages. Could you instead change `showShinyClientMessage` so that right before performing the append, see if the message already exists? (Or have `showShinyClientMessage` call a method on ShinyErrorConsole that tracks all of this)

Reply:
I updated this to use a static method `ShinyErrorConsole.appendConsoleMessage()` to handle adding the message, skipping appending if the message already exists. 48d07c55f15e5e362dab257ab91f8f699f7668a5